### ISSUE TITLE
Answers.logLoginWithMethod is not a function

### ISSUE THREAD

**Issue by richardbenson** [hypery2k/nativescript-fabric#108]
## Summary

> file:///app/tns_modules/nativescript-fabric/fabric.js:36:30: Unknown logging login in Fabric TypeError: Answers.logLoginWithMethod is not a function. (In 'Answers.logLoginWithMethod(method, success, null)', 'Answers.logLoginWithMethod' is undefined)

After calling `Fabric.logLogin('token', true);` in iOS simulator.

## Version information

```
All NativeScript components versions information
┌──────────────────┬─────────────────┬────────────────┬─────────────┐
│ Component        │ Current version │ Latest version │ Information │
│ nativescript     │ 3.2.1           │ 3.2.1          │ Up to date  │
│ tns-core-modules │ 3.2.0           │ 3.2.0          │ Up to date  │
│ tns-android      │ 3.2.0           │ 3.2.0          │ Up to date  │
│ tns-ios          │ 3.2.0           │ 3.2.0          │ Up to date  │
└──────────────────┴─────────────────┴────────────────┴─────────────┘
```

```
nativescript-fabric@1.0.6
```

## More Detail

Using Angular app, Crashlytics seems to be working as does generic stat recording, I just can't call the specific event methods.

At the top of the component I have `import { Fabric } from 'nativescript-fabric';` then call the `logLogin` as above

Let me know if you need anything else from me, thanks!

NB: @next is pointing to an older version at the moment, tagged as 1.0.5-develop-109 and the problem also exists there.

**Comment by hypery2k** [comment]
could you please check which Pods are installed in the iOS project?

**Comment by richardbenson** [comment]
Bear in mind, I am not very familiar with XCode, I think what you need is this?

![image](https://user-images.githubusercontent.com/98186/31219807-5b4aa07a-a9b6-11e7-8d81-ecad86f62130.png)

If you need anything else, just let me know.

**Comment by Spenstar** [comment]
I am also getting a typing error when running any of the function calls.
```console
CONSOLE ERROR file:///app/tns_modules/nativescript-fabric/fabric.js:46:30: Unknown logging content view in Fabric TypeError: Answers.logContentViewWithName is not a function. (In 'Answers.logContentViewWithName(name, type, id, null)', 'Answers.logContentViewWithName' is undefined)
```

@hypery2k Looking through your source code in the installed npm package, it looks like the typings for iOS are not installed/loaded, resulting in typing errors.

For example, in `fabric.appdelegate.d.ts` the `/// <reference path="node_modules/tns-platform-declarations/ios/ios.d.ts" />` is unable to load because it doesn't exist in the plugins `node_modules` directory.

**Comment by hypery2k** [comment]
fixes with #115
